gemrb/core/GUI/GameControl.cpp

```diff
@@ -2038,6 +2034,31 @@ bool GameControl::HandleActiveRegion(InfoPoint *trap, Actor * actor, const Point
 	return false;
 }
 
+// Calculate the angle between a clicked point and the first selected character,
+// so that we can set a sensible orientation for the formation.
+void GameControl::InitFormation(const Point &clickPoint)
+{
+	if (isFormationRotation) return;
+
+	// Of course single actors don't get rotated, but we need to ensure
+	// isFormationRotation is set in all cases where we initiate movement,
+	// since OnMouseUp tests for it.
+	if (core->GetGame()->selected.size() == 0) {
+		isFormationRotation = false;
```

Comment:
this can't possibly be true given the check on line 2041. you could just combine the 2 checks.